language/transaction-builder/generator/src/rust.rs

```diff
@@ -52,7 +52,7 @@ fn output_preamble(out: &mut dyn Write, local_types: bool) -> Result<()> {
 
 // This file was generated. Do not modify!
 //
-// To re-generate this code, run: `(cd language/stdlib && cargo run --release)`
```

Comment:
does stdlib tool now work outside of language/stdlib? it breaks for me otherwise (though my branch is several days off master)

Reply:
I thought I tried but let me try again (and change the cwd in the test command)

Reply:
problem solved